src/components/store/Diamonds/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './style.module.scss';
+
+const numberFormat = new Intl.NumberFormat('en-US');
+
+interface DiamondsProps {
+  count: number;
+  className?: string;
+}
+
+const Diamonds = ({ count, className }: DiamondsProps) => {
+  return (
+    <span className={className}>
+      {numberFormat.format(count)}
+      <span className={styles.diamond}>diamonds</span>
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/30709485/859012e9-3c5e-42a6-8b8a-7cad001a3385)

Nit: the invisible "diamonds" text becomes visible when highlighted! Not a huge deal, but you can fix it by targeting `.diamond::selection` if you want.